searchlib/src/vespa/searchlib/attribute/i_document_weight_attribute.cpp

```diff
@@ -13,7 +13,7 @@ class StringAsKey final : public IDocumentWeightAttribute::LookupKey {
 
     vespalib::stringref asString() const override { return _key; }
 private:
-    vespalib::string _key;
+    vespalib::stringref _key;
```

Comment:
it might actually be better to keep the string, to avoid the 0-terminated stringref requirement to escape the LookupKey API and into the attribute API...

Reply:
done